documentation/design/input.md

```diff
@@ -4,6 +4,8 @@ Github audit plugin will be using config.yaml file for taking input from user
 
```

Comment:
instead of "documentation", let's call the folder "docs"

Reply:
done